microsoft/PowerToys
# [PT Run]Removed BinaryFormatter

## Summary of the Pull Request
Replaced insicure BinaryFormatter.


## PR Checklist

- [x] **Closes:** #15610


## Detailed Description of the Pull Request / Additional comments
Replaced the BInaryFormatter with the already existing JsonStorage.


## Validation Steps Performed
- Launch PT Run from the PT Runner
- File `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Settings\ImageCache.json` should be created
- Perform some searches in PT Run
- Close PT
- File `%LOCALAPPDATA%\Microsoft\PowerToys\PowerToys Run\Settings\ImageCache.json` should be updated with cached images
